addOns/network/network.gradle.kts

```diff
@@ -60,6 +68,7 @@ spotless {
 }
 
 dependencies {
+	zapAddOn("brk")
```

Comment:
Hi @thc202 could you please help me with one error. I am getting the error as the following 

```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/aryangupta701/Desktop/contributions/zap-extensions/addOns/network/network.gradle.kts' line: 71

* What went wrong:
Project with path ':addOns:brk' could not be found in project ':addOns:network'.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
```